Multiple task v2 - FIX

yassirog opened the pull request:
## Intro
I created a new multiple-task page that includes all four mourning tasks. The old version didn't work well according the client, so I had to delete it completely and start over. For the desktop version, I mimicked the Figma sketch exactly, so the design was completely accurate. I sketched and worked out the mobile version myself, because there was no specific sketch for that.

## Visuals

### Desktop
![image](https://github.com/user-attachments/assets/28cedff2-0e2a-40e4-8d2a-89d2748c8781)
### Mobiel
![image](https://github.com/user-attachments/assets/33238f17-7675-4876-8621-ce33b8b89ea3)

## Info
I have this page the same for all interfaces (themes) except the colors and information for each theme. 

3 components made.

Header:
This is the header through which you can navigate to other pages within the project. 

Card:
These are the exercises you can click on at your interface.

Task selector:
This is the navigation between interfaces.

## Tests
How has this been tested?
- [ x ]  usertest
- [ ]  browser test
- [ ]  device test
- [ ]  screenreader test
- [ x ]  tab test
- [ x ]  performance test
- [ x ]  accessibility test
- [ x ]  javascript test

### Usertest
I had my brother test by navigating him through the first multiple-task-1 page to the other pages. This was successful so that means it is user-friendly since he had zero knowledge of the project.

### Tab test
I did the tab test myself and when I tab it goes through every element you should be able to click on so that's good. So that test also went well.

### Performance test
I tried to add Svelte enhanced images, but it wasn't successfull. I want to try again in the next coming days.
![image](https://github.com/user-attachments/assets/3c68608a-cc5a-4fd7-a1b9-0db3eb552d12)

### Accessibility test
I didn't use h1, h2 etc. I used h1 to h3 which is pretty stupid of me. That's why it gave me a 98 instead of a 100.
![image](https://github.com/user-attachments/assets/47ed4cef-069b-4e9c-a812-465f28335752)

### Javascript test
I also tested the javascript, i turned off the javascript to make sure my website is also good for people with devices where javascript isn't working. This test was successfull, my nav witht the arrow is made off select, so it always works and my cards are with javascript normally, but if the user has no javascript then all cards are automatically open.

yassirog committed:
Multi-task pages done

Multiple task pages af inclusief componenten:

Header
Card
Task-Selector

yassirog committed:
lazy loading added

lisavanmansom commented:
Hi, de pull request is netjes! Ik heb naar je code gekeken, zou je naar de volgende punten kunnen kijken en verbeteren voordat het wordt gemerged.

**Task-selector.svelte:**
- Je gebruikt de root in een component, die custom properties staan ook in de global.css, probeer die te gebruiken;
- Je Task-selector is niet PE, hiervoor zou je ook een details element kunnen gebruiken;
- Kijk naar je html elementen, je gebruikt nu veel div’s;

**Header.svelte:**
- Je kan van SVG’s beter een component maken zodat wanneer je ze toepast de code overzichtelijke blijft;
- Je Header is niet PE;
- In CSS maak je gebruik van pixel units, em units zijn beter.

**Card.svelte:**
- Je hebt unused css selectors in je code, ‘html, body’, hierdoor kan de site niet live aangezien vercel dit ziet als errors
- In CSS maak je gebruik van pixel units, em units zijn beter.

**Multiple-task-pages:**
- Hoe je files import is niet heel overzichtelijk, dit kan gewoon in 1 regel als je gebruik maakt van de ‘index.js’, voorbeeld: `import {BlurGradient, QuestionForm} from '$lib’`, is hoe het er dan uit kan zien;
- Je kan beter voor 4 dezelfde pagina’s een slug gebruiken (idk of je hier tijd voor hebt?);

**Static/gradients -> gr-purple.png:**
- Dit hoort een div te zijn, is ook beter voor performance als het een div is, dit was vrijdag tijdens de CD besproken en maandag ook tijdens de standup (wegens omstandigheden was je toen niet aanwezig, maar het staat wel gedocumenteerd, het component is hiervoor al gemaakt, die kan je dan toepassen)

**Algemene feedback:**
- In je pull request heb je het over de heading levels niet correct zijn, pas dit dan aan voordat je een pull request maakt, want het is wel fijn als dit nu alsnog wordt verbeterd

yassirog committed:
PE and other issues fixed

Most issues fixed on the feedback of Lisa

SVG's to components
px to EM
h1 to h2
unused selectors deleted
div's changed to other elements
root deleted
